main_routers/config_router.py

```
@@ -564,3 +564,107 @@ async def get_api_providers_config():
         }
 
 
+@router.post("/gptsovits/load_model")
+async def load_gptsovits_model(request: Request):
+    """代理请求到 GPT-SoVITS API 加载模型（解决 CORS 问题）"""
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)